tap_s3_csv/__init__.py

```diff
@@ -47,6 +47,11 @@ def do_sync(config, catalog, state):
     json_lib = config.get('json_lib', 'orjson')
     row_limit = config.get('row_limit', None)
 
+    # Exports splunk logs for row and col count
```

Comment:
I think its not specific to splunk, we are just logging it. Splunk just happens to be end product making that logs useful.